include/dos_inc.h

```diff
@@ -809,22 +809,51 @@ enum COUNTRY {
 	Norway = 47,
 	Poland = 48,
 	Germany = 49,
+	Argentina = 54,
 	Brazil = 55,
+	Malaysia = 60,
 	Australia = 61,
+	Philippines = 63,
+	Singapore = 65,
+	Kazakhstan = 77,
 	Japan = 81,
 	South_Korea = 82,
+	Vietnam = 84,
 	China = 86,
```

Comment:
Not sure if this is just a github rendering thing, or if the new block needs one more tab inward?

Here's what I'm seeing:

![2022-03-29_16-58](https://user-images.githubusercontent.com/1557255/160725086-719d14f8-d600-4f6d-b5d4-7d6726ac5a4f.png)